lms/static/styles/components/_Dialog.scss

```diff
@@ -26,10 +20,13 @@
 }
 
 .Dialog__content {
+  display: flex;
+  flex-direction: column;
+  max-height: 90vh;
```

Comment:
These rules mix a `max-height` relative to the viewport and fixed `max-width`. Is it feasible to make `max-width` proportional, as well? What happens with the `700px` max-width if the viewport is narrower than 700px?

Reply:
Perhaps, and I'll test. If the content pushes this out really wide, it might be harder to read.

Reply:
Also, it will shrink down nicely if content pushes out to 700 but then the window goes more narrow. Everything will wrap down to the minimal amount the content can wrap to.